core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -81,9 +76,7 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
 {% endblock %}
 
 {% block footer %}
-  {% if not iframed %}
-    <exploration-footer twitter-text="{{DEFAULT_TWITTER_SHARE_MESSAGE_PLAYER}}"></exploration-footer>
-  {% endif %}
+  <exploration-footer ng-if="!iframed" twitter-text="{{DEFAULT_TWITTER_SHARE_MESSAGE_PLAYER}}"></exploration-footer>
```

Comment:
I understand that the use of Jinja here was not noted in the original issue, but could you take care of it as well please (so that we can fully close this file)? I think you'll be dealing with it the same way as explorationVersion.